safe/gui/tools/wizard/step_kw57_extra_keywords.py

```diff
@@ -127,8 +135,10 @@ def set_widgets(self):
             # Timezone
             timezone_checkbox = QCheckBox()
             timezone_combo_box = QComboBox()
-            for timezone in pytz.common_timezones:
-                timezone_combo_box.addItem(timezone, timezone)
+            for timezone in extra_keyword_time_zone['options']:
+                timezone_combo_box.addItem(
+                    timezone['key'], timezone['name']
+                )
             index = timezone_combo_box.findText('Asia/Jakarta')
```

Comment:
Maybe you can move it to definitions? I like the way you did `options`. `default_option` ? Feel free to skip this request